db/osqlcomm.c

```diff
@@ -3294,7 +3298,7 @@ int osql_comm_send_poke(char *tohost, unsigned long long rqid, uuid_t uuid,
         poke.tstamp = comdb2_time_epoch();
 
         poke.from = gbl_mynodeid;
-        poke.to = nodenum(tohost);
+        poke.to = nodenum((char *)tohost);
```

Comment:
Can nodenum() be changed to take a const char * parameter?

Reply:
unrelated to this PR